app/controllers/application_controller.rb

```diff
@@ -7,9 +7,7 @@ class ApplicationController < ActionController::Base
   private
 
   def login_required
-    if session[:user_id]
-      @current_user = User.find(session[:user_id])
-    end
+    redirect_to root_path, notice: "ログインしてください" unless current_user
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)